docs/website/docs/reference/frequently-asked-questions.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Frequently Asked Questions
+description: Questions asked frequently by users in technical help or github issues
+keywords: [faq, usage information, technical help]
+---
+
+
+#### Q: Can I configure different nesting levels for each resource in a source within DLT?
+
+Currently, configuring different nesting levels for each resource directly is not supported, but there's an open GitHub issue ([#945](https://github.com/dlt-hub/dlt/issues/945)) addressing this. Meanwhile, you can use these workarounds:
+
+Separate Resources Based on Nesting Needs:
+Separate the execution of pipelines for resources based on their required maximum table nesting levels. For example:
+
+For resources that don't require nesting (resource1, resource2):
```

Comment:
Why all the paragraphs end with a colon mark?

Reply:
removed unnecessary ones